processor/src/test/java/com/linecorp/decaton/processor/runtime/internal/AbstractSubPartitionsTest.java

```diff
@@ -64,61 +55,27 @@ private static PartitionScope scope(
                 new TopicPartition(topic, 0));
     }
 
-    @Mock
-    private Processors<?> processors;
-
-    @Test
-    public void testCleanupPartiallyInitializedUnits() throws Exception {
```

Comment:
Because we no longer initialize processor units and instead do it lazily: https://github.com/line/decaton/pull/224/files#diff-c68a7fb9de5afb7968afda54f084633b7d5a38ec068e84fcad000d582ec5eaafR77